src/backends/gitlab.rs

```diff
@@ -63,12 +63,19 @@ impl Release {
 /// `ReleaseList` Builder
 #[derive(Clone, Debug)]
 pub struct ReleaseListBuilder {
+    server: String,
     repo_owner: Option<String>,
     repo_name: Option<String>,
     target: Option<String>,
     auth_token: Option<String>,
 }
 impl ReleaseListBuilder {
+    /// Set the gitlab `server` url
+    pub fn use_server(&mut self, server: &str) -> &mut Self {
```

Comment:
Could you rename this to with_host instead of server?

Reply:
Done